src/detect-engine-mpm.c

```diff
@@ -643,6 +643,12 @@ uint16_t PatternMatchDefaultMatcher(void)
             }
         }
 
+#ifndef BUILD_HYPERSCAN
+    if (strcmp(mpm_algo, "hs") == 0) {
+        FatalError(SC_ERR_INVALID_VALUE, "Hyperscan (hs) support is "
```

Comment:
this error provides no guidance or even enough context for users with little experience. Would suggest adding that it is about the mpm-algo option value.

Reply:
> this error provides no guidance or even enough context for users with little experience. Would suggest adding that it is about the mpm-algo option value.

That's correct. I ran it in dev mode and could see the file names so missed it.

How is `"Hyperscan (hs) support for mpm-algo is not compiled into Suricata"`?

Reply:
Add "Add`-with-libhs-includes and -with-libhs-libraries to configure."?